src/language.rs

```diff
@@ -0,0 +1,260 @@
+        //expect text
+        create_temp_dir_expect: "Couldn't create the temporary directory!",
+        remove_temp_dir_expect: "Couldn't remove the temporary directory!",
+        
+        //=====# posters.rs translations #=====//
+        //err text
+        msg_err: "Couldn't create image in",
+        reading_err: "Error Reading",
+        downloading_err: "Error Downloading",
+    
+        //=====# choose_episode.rs translations #=====//
+        //misc text
+        total_episode_misc_text: "Total of episodes to watch:",
+        select_episode_misc_text: "Select the episode you want watch:",
+        //err text
+        choose_episode_err: "There was an error, please try again",
+    
+        //=====# choose_lang.rs translations #=====//
+        //misc text
+        select_lang_misc_text: "Select the language option:",
+        //err text
+        choose_lang_err: "There was an error, please try again",
+    
+        //=====# choose_media.rs translations #=====//
+        //misc text
+        total_media_misc_text: "Total of media to watch:",
+        select_media_misc_text: "Select what you want to watch:",
+        //err text
+        choose_media_err: "There was an error, please try again",
+    
+        //=====# choose_season.rs translations #=====//
+        //misc text
+        total_season_misc_text: "Total of seasons to watch:",
+        select_season_misc_text: "Select the season you want watch:",
+        //err text
+        choose_season_err: "There was an error, please try again",
+    
+        //=====# get_medias.rs translations #=====//
+        //expect
+        response_expect: "Could not load url.",
+    
+        //=====# vlc.rs translations #=====//
+        //misc text
+        vlc_start_misc_text: "Starting the player",
+        vlc_exit_misc_text: "Bye bye (^_^)/",
+        // err text
+        vlc_exit_with_err: "VLC exited with an error:",
+        vlc_wait_err: "Failed to wait for VLC:",
+        vlc_start_err: "Failed to start VLC:",
+    
+        //=====# watch_media.rs translations #=====//
+        //misc text
+        preparing_misc_text: "Preparing everything, which can take a while",
+        getting_episodes_misc_text: "Getting episodes",
+        getting_language_misc_text: "Getting languages options",
+        fetching_misc_text: "Fetching service",
+        //err text
+        click_season_err: "Error: Can't click on the season",
+        click_episode_err: "Error: Can't click on the episode",
+        //expect text
+        language_option_expect: "Couldn't retrieve languages options.",
+    };
+
+        return english_translation;
+}
+
+pub fn choose_language() -> Translations {
+    let use_english = false;
+
+        if use_english {
+            let english_translation = english();
+            return english_translation;
+        }
+        else {
+            let portuguese_translation = portuguese();
+            return portuguese_translation;
+        }
+}
```

Comment:
Instead of using a function to determine the language, perhaps we could create an argument called `english` with the short flag `en` to allow the user to change the language

Reply:
for what i searched in clap short flags can't be more than one character

Reply:
added argument to determine the language, but the way that is implemented is : 

when the argument is passed it generate one bool that is used for the "choose_language()" function to determine if it will use English or Portuguese, the bad thing about this method is that the bool need to be passed as one argument of the function for each time "choose_language()" is called.

if you think that this method is not good enough let me know that i will try to find a better way to do it

Reply:
Maybe instead of creating a new function and a new variable, which time we choose a language, we can create only one variable called `language` that by default has the value of the function `portuguese`, and when choosing the argument `english` we make the variable `language` equal to the function `english`

Reply:
changed in [this commit](https://github.com/anotherlusitano/vizer-cli/pull/11/commits/eb355d7261ee69c14dcfc74a4d03cbb6709cdc67). if has a better way to do that, just let me know, that i will try to find it out

Reply:
Its great! 
The only issue here is that there are too many `unsafe` blocks, but I confess that I'm still learning Rust and I don't know other way to do it :sweat_smile: 
Maybe [this post](https://www.sitepoint.com/rust-global-variables/) can help?
Tell me what you think :)

Reply:
lmaoo yeah, this post is great i got excited and tried alot of differentes way still no success lmao,

**list of what i tried**
- only Mutex / doesn't work
why it fails =  "it's extremely trick to manage the mutex guard while needing to change it every time"

- lazy_mut / doesn't work

- lazy_static / doen't work 
 (but i think, i don't try every way possible, if the others options fail i will come back to this one)

- RefCell and thread_local() doesn't work 
why it fails = "RefCell and thread_local get the variable and change it only inside the thread that is created, if i change the struct to English, it will change only inside the currently thread"

i'm currently trying Mutex + BorrowMut and it's looking that it will be the way

Reply:
Keep trying and don't worry if you didn't find a solution; we can merge this after you finish and look for a better solution later :)

Reply:
:) i see, well good idea, none of the methods worked for me, i can try it again later but for now I'm done with that fix lmao.

i managed to reduce the numbers of unsafe by send the entire struct to one variable instead of calling the struct for each string, but still is not the ideal, i think.

i don't know if this pull request is ready to be merged, because still has some bugs like
- help option not working with arguments
 (like "--help --search" not displaying the specific help of the search flag)

- help option not being translated with the --english flag

Reply:
Lets just make the help option be in english to make it easy

Reply:
done :), no more known issues,
removing the translation from the --help option also fixes the "help option not working with arguments". 

now is just the unsafe caveat.

Reply:
**update**, now the code uses only 1 unsafe statement, it's good enough or the code refactoring with others method is still planed?

Reply:
good job